common/tensorrt_common/CMakeLists.txt

```diff
@@ -1,57 +1,81 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.17)
```

Comment:
Why does it need to version up to 3.17?

Reply:
`FindCUDA` is deprecated since cmake 3.10.
https://cmake.org/cmake/help/latest/module/FindCUDA.html
I used `FindCUDAToolkit` which is available since cmake 3.17 instead of `FindCUDA`.
https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html

Reply:
okay
I think there is no problem since cmake version 3.22.1 is currently installed.
cc @xmfcx @kenji-miyake